pkg/api/types.go

```diff
@@ -145,9 +145,20 @@ func (r Resources) ValidateNonZero() error {
 	return nil
 }
 
+func (r Resources) SanityCheck() error {
+	if r.VCPU.Cmp(*resource.NewMilliQuantity(250, resource.BinarySI)) == -1 {
```

Comment:
IMO 0.25 is too big here. I think the big thing we're worried about here is the "accidentally multiplied or divided by 1000", and it's possible we want less than 0.25 CPU in the future. I'd use 0.05 instead (x1000 would be 50 CPUs, so this should have few false-positives and only has false-negatives on really big VMs)

cc @tychoish for second opinion here